src/test/java/com/epam/DressesSubMenuTest.java

```diff
@@ -0,0 +1,29 @@
+package com.epam;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class DressesSubMenuTest {
+
+    @Test
+    @DisplayName("Verify Dresses tab has correct sub-menu items")
+    public void dressesSubMenu() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.get("http://automationpractice.com/index.php");
+
+        WebElement dressesTab = driver.findElement(By.xpath("//div[@id='block_top_menu']/ul[@class]/li/a[@title='Dresses']"));
```

Comment:
this was tricky. you found potentially the best locator.
however, don't forget that it's still a weak locator and the only thing we could still try is to change the application code. if it's an option to have access to it, we should definitely try!